ShowsLibrary/ShowsLibrary/Frequency.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+namespace ShowsLibrary
+{
+    public enum Frequency
+    {
+        daily,
+        weekly,
+        monthly,
+        nonPeriodic
+    }
```

Comment:
Стиль! Имена в перечислениях обычно пишут с большой буквы
```suggestion
    public enum Frequency
    {
        Daily,
        Weekly,
        Monthly,
        NonPeriodic
    }
```